### PR TITLE
Disassemble more dungeon functions

### PR DESCRIPTION
**Pull request by chordtoll**

### CODE REVIEW

File: src/overlay_29_02338604.c [hunk 1]
```diff
@@ -4,6 +4,21 @@
...
+u8 IsSecretBazaar()
+{
+    return DUNGEON_PTR[0]->gen_info.fixed_room_id==FIXED_SECRET_BAZAAR;
+}
+
+
+u8 ShouldBoostHiddenStairsSpawnChance()
+{
+    #ifdef JAPAN
+    return DUNGEON_PTR[0]->boost_hidden_stairs_spawn_chance;
```
Comment (AnonymousRandomPerson) [inline src/overlay_29_02338604.c:16]: This is the same between Japan and other regions, so the `#ifdef` isn't needed.

**Review by AnonymousRandomPerson — CHANGES_REQUESTED** [review]
Looks like the Japan build doesn't match, which is causing the build failure.

**Comment by AnonymousRandomPerson** [comment]
I looked into the JP mismatch, and it's because we aren't yet sure where the dungeon struct differs between JP and other regions. Here's a commit you can use to match JP for now: https://github.com/AnonymousRandomPerson/pmd-sky/commit/0ef94e7ad80e23101ae5cf22554f5c0e2d4cd64e

**Comment by chordtoll** [comment]
Thanks! That helps a lot! I'll keep in mind that we might need to move those ifdefs around in the future, and have updated my pre-pr build to build all languages.

**Comment by AnonymousRandomPerson** [comment]
Build is failing because of Wine not downloading properly, possibly due to mirrors being out of sync. This happens on occasion and should resolve itself in a while.

**Review by AnonymousRandomPerson — APPROVED** [review]

**Pull request merged**